Sources/LanguageServerProtocol/Language.swift

```diff
@@ -38,12 +33,22 @@ public enum Language: String, Codable, Hashable {
       case .cpp: return "c++"
       case .objective_c: return "objective-c"
       case .objective_cpp: return "objective-c++"
-      case .unknown: return nil
+      default: return nil
     }
   }
 
   /// Clang-compatible language name for a header file. See `xflag`.
   public var xflagHeader: String? {
     return xflag.map { "\($0)-header" }
   }
+
+  public static func ~= (lhs: Language, rhs: Language) -> Bool {
```

Comment:
Why is this needed? I would have expected being Equatable was enough.

Reply:
not needed